tests/pyoozie/test_tags.py

```diff
@@ -237,56 +233,56 @@ def test_global_configuration():
 
 def test_email():
     actual = Email(
-        to='mrt@theateam.com',
+        to='mrt@example.com',
         subject='Chains',
         body='Do you need more?',
     ).xml(indent=True)
     assert xml_to_dict_unordered('''
     <email xmlns="uri:oozie:email-action:0.2">
-        <to>mrt@theateam.com</to>
+        <to>mrt@example.com</to>
         <subject>Chains</subject>
         <body>Do you need more?</body>
     </email>
     ''') == xml_to_dict_unordered(actual)
 
     actual = Email(
-        to='mrt@theateam.com',
+        to='mrt@example.com',
         subject='Chains',
         body='Do you need more?',
         cc='ateam@ateam.com',
```

Comment:
Should probably hit the `cc`s too for consistency. 
Subdomains for colour maybe? (e.g. `mrt@theateam.example.com`) I'd be a shame to ruin perfectly amusing testcases.